molecule/openstack/molecule.yml

```diff
@@ -0,0 +1,29 @@
+dependency:
+  name: galaxy
+driver:
+  name: openstack
+lint:
+  name: yamllint
+  options:
+    config-file: tests/yamllint.yml
+platforms:
+  - name: test-passwd
+    image: rhel-7.4-server-x86_64-updated
+    flavor: m1.medium
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  playbooks:
+    prepare: ../shared/prepare.yml
+    converge: ../shared/playbook.yml
+  inventory:
+    links:
+      group_vars: ../shared/group_vars
```

Comment:
This is breaking travis; there are no group_vars in this PR.

In general, I've come to dislike using shared group_vars, since it becomes a minor pain to keep up with the vars and the playbook during testing. I think DRY is not always appropriate for testing, and it's probably fine to include var definitions in the converge/prepare playbooks that use them for the sake of readability.